src/components/RadioGroupField/RadioGroupField.tsx

```diff
@@ -23,13 +23,17 @@ const styles = createStyles({
     "& > span": {
       padding: "6px"
     }
+  },
+  secondLabel: {
+    display: "block",
+    fontSize: "12px"
   }
 });
 
-interface RadioGroupFieldProps extends WithStyles<typeof styles> {
+interface RadioGroupFieldProps {
   choices: Array<{
```

Comment:
Not changed in this PR, but it would be better if it would be named type instead of literal wrapped in `Array<T>`